src/+datools/+statistical/+ensemble/EnKF.m

```diff
@@ -30,20 +30,21 @@ function analysis(obj, R, y)
                 rhoHt = ones(size(Af, 1), size(HAf, 1));
                 HrhoHt = ones(size(HAf, 1), size(HAf, 1));
             else
-                H = obj.Observation.linearization(tc, xfm);
+                H = obj.ObservationOperator.linearization(tc, xfm);
                 rhoHt = obj.Localization(tc, xfm, H);
                 HrhoHt = H * rhoHt;
             end
 
             PfHt = rhoHt .* ((1 / (ensN - 1)) * (Af * (HAf.')));
             HPfHt = HrhoHt .* ((1 / (ensN - 1)) * (HAf * (HAf.')));
 
-            S = HPfHt + R;
+            S = HPfHt + obj.Observation.R;
             dS = decomposition(S, 'chol');
-            d = y - Hxfm;
+            d = obj.Observation.Y - Hxfm;
 
             xam = xfm + PfHt * (dS \ d);
-            Aa = Af + PfHt * (dS \ (sqrtm(R) * randn(size(HAf)) - HAf));
+            Aa = Af + PfHt * (dS \ (sqrtm(obj.Observation.R) *...
```

Comment:
Why is this on two lines? 80 character lines are far too small. This isn't 1985. 

Set the limit to something reasonable like 120.